helm-chart/renku-notebooks/Chart.yaml

```diff
@@ -2,4 +2,4 @@ apiVersion: v1
 appVersion: '1.0'
 description: A Helm chart for the Renku Notebooks service
 name: renku-notebooks
-version: 0.5.2
+version: 0.0.1-set.by.chartpress
```

Comment:
with this change don't we always get `0.0.1-<8-digit-commit-sha>`. Do we want this? :thinking:

Reply:
no definitely not, I had a wrong chartpress version installed